packages/util/folders.go

```diff
@@ -27,14 +27,6 @@ func GetAllFolders(params models.GetAllFoldersParameters) ([]models.SingleFolder
 			loggedInUserDetails = EstablishUserLoginSession()
 		}
 
-		if params.WorkspaceId == "" {
```

Comment:
Get all folders doesn't need to do these checks.

All commands use `GetWorkspaceConfigFromCommandOrFile` and perform this check in advance.